### PR TITLE
add OpenBSD 6.2

### LINKED ISSUE #55: install bsd.mp on all OpenBSD boxes
we will need multi-processors kernel. all box has `bsd` only.

### PR DESCRIPTION
**Pull request by trombik**
this PR add OpenBSD 6.2.

also:

* OpenBSD boxes fetche packages from one of the official CDN
* install `bsd.mp`, fixes #55 

as a result, all OpenBSD boxes will be updated

### CODE REVIEW

File: scripts/openbsd/init.sh [hunk 1]
```diff
@@ -3,9 +3,12 @@
 set -e
 set -x
 
-sudo tee /etc/pkg.conf <<EOF
-installpath = ftp.openbsd.org
+# pkg.conf has been replaces with installurl since 6.1
```
Comment (mitsururike) [inline scripts/openbsd/init.sh]: typo: s/replaces/replaced/

File: scripts/openbsd/init.sh [hunk 2]
```diff
@@ -42,7 +45,7 @@ fi
 # XXX this patch should be applied in our ports tree, but we desparately need
 # package builder first. remove this when the builder is ready.
 #
-if [ `ansible --version | head -n 1 | cut -d' ' -f 2` == '2.3.2.0' ]; then
+if [ `ansible --version | head -n 1 | cut -d' ' -f 2` == '2.3.2.0' -a `uname -r` != '6.2' ]; then
```
Comment (mitsururike) [inline scripts/openbsd/init.sh]: ansible package is installed from our ports tree on `6.2` and the patch is necessary.
or exclude `6.2` before the line 15 `sudo pkg_delete ansible`?

Comment (mitsururike) [inline scripts/openbsd/init.sh]: our ports tree has been patched for `6.2`?

File: reallyenglish_spec/packages_spec.rb [hunk 3]
```diff
@@ -15,24 +15,27 @@
   end
 when "openbsd"
   # XXX RE_5_9 does not have the latest ansible yet
-  if os[:release].to_f >= 6.0
+  if os[:release].to_f >= 6.0 && os[:release].to_f < 6.2
```
Comment (mitsururike) [inline reallyenglish_spec/packages_spec.rb]: i do not understand why 6.2 and above are excluded.

File: scripts/openbsd/init.sh (hunk 2 above)
Comment (trombik) [reply scripts/openbsd/init.sh]: have not looked at the `patches` directory yet, but the package in `OPENBSD_6_2` has passed specs that check the bug in question.

**Comment by trombik** [comment]
all boxes have passed the tests.

**Comment by trombik** [comment]
all boxes have passed the tests.

**Review by mitsururike — COMMENTED** [review]
tests in a box booted with multiple CPUs should be added to ensure that the installed kernel can boot single and multi CPU box and uses all the CPUs.

**Comment by trombik** [comment]
i have been working on another feature branch. the tests you have mentioned would be implemented in the branch

**Review by mitsururike — APPROVED** [review]

**Comment by trombik** [comment]
f71aa2c adds the tests

**Pull request merged**